wagtail/admin/views/reports.py

```diff
@@ -28,3 +29,8 @@ def get_queryset(self):
         pages = UserPagePermissionsProxy(self.request.user).editable_pages().filter(locked=True)
         self.queryset = pages
         return super().get_queryset()
+
+    def dispatch(self, request, *args, **kwargs):
+        if not UserPagePermissionsProxy(request.user).can_unlock_pages():
```

Comment:
This should now be `can_remove_locks`. I guess the fact that this wasn't caught by tests means that we're lacking test coverage for this view too...

Reply:
Good point, thanks. I've fixed this and added a couple of basic tests (accessing it with no locked pages, locking a page)